zproject/backends.py

```diff
@@ -638,7 +638,9 @@ def authenticate(self, dev_auth_username: Optional[str]=None, realm: Optional[Re
         return common_get_active_user(dev_auth_username, realm, return_data=return_data)
 
 def redirect_deactivated_user_to_login() -> HttpResponseRedirect:
-    login_url = reverse('django.contrib.auth.views.login')
+    # Specifying the template name makes sure that the user is not redirected to dev_login in case of
+    # a deactivated account on a test server.
+    login_url = reverse('zerver.views.auth.login_page', kwargs = {'template_name': 'zerver/login.html'})
```

Comment:
Redirecting to `/login` made testing easier for me, please let me know if the deactivated user needs to be redirected to `/dev_login` instead.

Reply:
I think /login is better in the development environment, since that's the page that can show an error message.